internal/host/installcmd_test.go

```diff
@@ -76,16 +76,19 @@ var _ = Describe("installcmd", func() {
 	Context("negative", func() {
 		It("get_step_one_master", func() {
 			mockValidator.EXPECT().GetHostValidDisks(gomock.Any()).Return(nil, errors.New("error")).Times(1)
+			mockVersions.EXPECT().GetReleaseImage(gomock.Any()).Return("releaseImage").Times(1)
```

Comment:
maybe it should be a helper function? this way if the api changes you have only one place to change
```go
mockGetReleaseImage := func () { mockVersions.EXPECT().GetReleaseImage(gomock.Any()).Return("releaseImage").Times(1)
}
```